pandas/core/base.py

```diff
@@ -355,6 +355,13 @@ def ndim(self) -> Literal[1]:
         """
         Number of dimensions of the underlying data, by definition 1.
 
+        See Also
+        --------
+        Index.size: Return the number of elements in the underlying data.
```

Comment:
For this file, could you reference `Series` instead of `Index?

Reply:
sure!